ms-campus-kid-school/src/utils/types.go

```diff
@@ -0,0 +1,20 @@
+package utils
+
+type Site struct {
+	Id_Site   int64  `json:"id_site"`
+	Site_Name string `json:"site_name"`
+	Site_Code string `json:"site_code"`
```

Comment:
Please follow the conventions for columns and variables naming we are using in the project.
```suggestion
	id int64  `json:"id"`
	name string `json:"name"`
	code string `json:"code"`
```